menas/ui/index.html

```diff
@@ -38,6 +38,8 @@
     <script src="components/dataset/conformanceRule/schemaManager.js"></script>
     <script src="components/dataset/conformanceRule/MappingConformanceRule/JoinConditionDialog.js"></script>
     <script src="components/dataset/conformanceRule/MappingConformanceRule/JoinConditionDialogFactory.js"></script>
+    <script src="components/dataset/conformanceRule/ConcatenationConformanceRule/ConcatenationColumnDialog.js"></script>
+    <script src="components/dataset/conformanceRule/ConcatenationConformanceRule/ConcatenationColumnDialogFactory.js"></script>
```

Comment:
I wish ES6 imports worked without these... 🙄

Reply:
They do if you use a real bundler, but UI5 doesn't play ball with ES6 or any transpiling bundler (#314). That's something I'll figure out how to make work in the future. Already wasted 2 todays on it a while ago.